src/components/bottom-sheet/BottomSheetContext.tsx

```diff
@@ -20,22 +21,20 @@ import {Backdrop} from './Backdrop';
 import {ClickableBackground} from './ClickableBackground';
 import {AnimatedBottomSheet} from './AnimatedBottomSheet';
 
-type BottomSheetContentFunction = (
-  close: () => void,
-  /** Use focusRef to give a component accessibility focus when BottomSheet open. The component must be accessible! */
-  focusRef: RefObject<any>,
-) => ReactNode;
+type BottomSheetContentFunction = () => ReactNode;
 
 type BottomSheetState = {
   open: (
     contentFunction: BottomSheetContentFunction,
-    /** Optional ref to component which should be focused on sheet close */
-    closeRef?: RefObject<any>,
     useBackdrop?: boolean,
   ) => void;
   isOpen: () => boolean;
   close: () => void;
   height: number;
+  /** Use onOpenFocusRef to give a component accessibility focus when BottomSheet open. The component must be accessible! */
+  onOpenFocusRef?: RefObject<any>;
+  /** Optional ref to component which should be focused on sheet close */
+  onCloseFocusRef?: RefObject<any>;
```

Comment:
Does these need to be typed as optional?

Reply:
Not needed I think, removed now